packages/mui-joy/src/Autocomplete/Autocomplete.tsx

```diff
@@ -200,6 +200,7 @@ const AutocompleteClearIndicator = styled(StyledIconButton as unknown as 'button
   slot: 'ClearIndicator',
   overridesResolver: (props, styles) => styles.clearIndicator,
 })<{ ownerState: OwnerState }>(({ ownerState }) => ({
+  alignSelf: 'center',
```

Comment:
This is required because the `Input`'s root no longer have `align-items: center`.